### PR TITLE
Update azure docs to include new param

### PR DESCRIPTION
**Pull request by Zlaticanin**
Very small PR to update azure docs to include new param `explicit_max_ttl`. 
PR: https://github.com/hashicorp/vault-plugin-secrets-azure/pull/199

### CODE REVIEW

File: website/content/api-docs/secret/azure.mdx [hunk 1]
```diff
@@ -216,6 +216,8 @@ information about roles.
   Accepts time suffixed strings ("1h") or an integer number of seconds. Defaults to the system/engine default TTL time.
 - `max_ttl` (`string: ""`) – Specifies the maximum TTL for service principals generated using this role. Accepts time
   suffixed strings ("1h") or an integer number of seconds. Defaults to the system/engine max TTL time.
+- `explicit_max_ttl` (`string: ""`) – Specifies the explicit maximum lifetime of the lease and service principal.
+  If not set or set to 0, will use the system default.
```
Comment (vinay-gopalan) [inline website/content/api-docs/secret/azure.mdx:220]: nit: is it worth mentioning what the system default is (10 years I believe)?

**Review by vinay-gopalan — APPROVED** [review]

File: website/content/api-docs/secret/azure.mdx (hunk 1 above)
Comment (Zlaticanin) [reply website/content/api-docs/secret/azure.mdx:220]: Good idea. I will update it!

File: website/content/api-docs/secret/azure.mdx [hunk 2]
```diff
@@ -216,6 +216,8 @@ information about roles.
   Accepts time suffixed strings ("1h") or an integer number of seconds. Defaults to the system/engine default TTL time.
 - `max_ttl` (`string: ""`) – Specifies the maximum TTL for service principals generated using this role. Accepts time
   suffixed strings ("1h") or an integer number of seconds. Defaults to the system/engine max TTL time.
+- `explicit_max_ttl` (`string: ""`) – Specifies the explicit maximum lifetime of the lease and service principal.
```
Comment (gsantos-hc) [inline website/content/api-docs/secret/azure.mdx:219]: How about:

> Limits the lifetime of the service principal secret in Azure and sets an upper bound on the corresponding lease's TTL. This lifetime cannot be changed once a lease is created, and the lease cannot be renewed beyond the expiration of this lifetime. If not set or set to zero, the service principal secret will be created with a default duration of 10 years and deleted once the corresponding lease in Vault expires.

**Review by gsantos-hc — COMMENTED** [review]
Does there need to be an update to the plugin version or is that handled separately?

**Comment by gsantos-hc** [comment]
Sorry – Looks like my comments posted after this was merged 🤷‍♂️ It was a small improvement anyway.

**Pull request merged**